File: backend/WarehouseOps.Domain/Product.cs
namespace WarehouseOps.Domain;

public class Product : BaseEntity
{
    public string Name { get; set; } = string.Empty;

    public string Description { get; set; } = string.Empty;

    public decimal Price { get; set; }

    public string Sku { get; set; } = string.Empty;
}

File: frontend/warehouseops-client/src/types/product.ts
export type Product = {
  id: string;
  name: string;
  sku: string;
  category: string;
  description: string;
  price: number;
  createdAt: string;
  updatedAt: string | null;
};

export type ProductFilters = {
  search?: string;
  category?: string;
};
